test/validationTest.lua

```diff
@@ -15,83 +20,257 @@ local json_hmi_tbl = { numTicks = 7, position = 6, sliderHeader ="sliderHeader",
 else
-  print("validate_mobile_response:"..tostring(_res).." ==> ".._err)
+  print("validate_hmi_response Buttons.GetCapabilities:"..tostring(_res).." ==> ".._err)
 end
 
-_res, _err = validator.validate_mobile_response( 'Slider',{ success = 'true', resultCode = {"SUCCESS"} },true)
+local _res, _err = hmi_schema:Validate('UI.Slider',  'request', json_hmi_tbl) 
+if _res then
+  print("validate_hmi_request:"..tostring(_res))
+else
+  print("validate_hmi_request:"..tostring(_res).." ==> ".._err)
+end
+
+
+_res, _err = mob_schema:Validate('Slider', 'response',  { success = true, resultCode = "SUCCESS" },true)
 if _res then
   print("validate_mobile_response:"..tostring(_res))
 else
   print("validate_mobile_response:"..tostring(_res).." ==> ".._err)
 end
 
-_res, _err = validator.validate_mobile_response( "WrongFunctionName", { success = false, resultCode = "INVALID_DATA", info = nil })
+
+_res, _err = mob_schema:Validate("WrongFunctionName", "response", { success = false, resultCode = "INVALID_DATA", info = nil })
 if _res then
-  print("validate_mobile_response with \"WrongFuntionName\":"..tostring(_res))
+  print("validate_mobile_response with \"WrongFunctionName\":"..tostring(_res))
 else
-  print("validate_mobile_response with \"WrongFuntionName\":"..tostring(_res).." ==> ".._err)
+  print("validate_mobile_response with \"WrongFunctionName\":"..tostring(_res).." ==> ".._err)
 end
 
-_res, _err = validator.validate_hmi_notification( 'BasicCommunication.OnSystemRequest', { requestType = {"PROPRIETARY"}})
+_res, _err = hmi_schema:Validate("BasicCommunication.OnSystemRequest", 'notification',  { requestType = "PROPRIETARY"})
 if _res then
   print("validate_hmi_notification:"..tostring(_res))
 else
   print("validate_hmi_notification:"..tostring(_res).." ==> ".._err)
 end
 
-_res, _err = validator.validate_hmi_notification( 'BasicCommunication.OnSystemRequest', { urlSchema = "default", fileName = "fileName"}, true)
+_res, _err = hmi_schema:Validate('BasicCommunication.OnSystemRequest', "notification", { url = "default", fileName = "fileName"}, true)
 if _res then
   print("validate_hmi_notification:"..tostring(_res))
 else
   print("validate_hmi_notification:"..tostring(_res).." ==> ".._err)
 end
 
-_res, _err = validator.validate_mobile_notification( "OnHMIStatus", {hmiLevel = "FULL"})
+_res, _err =mob_schema:Validate("PerformInteraction", "request",  { initialText = "initialText",
+    initialPrompt = {{text = "initialPrompt", type = "TEXT"}},
+    interactionMode = "MANUAL_ONLY",
+    interactionChoiceSetIDList = {1},
+    helpPrompt = {{text = "helpPrompt", type = "TEXT"}},
+    interactionLayout = "KEYBOARD"
+  })
 if _res then
-  print("validate_modile_notification:"..tostring(_res))
+  print("validate_mobile_request:"..tostring(_res))
 else
-  print("validate_mobile_notification:"..tostring(_res).." ==> ".._err)
+  print("validate_mobile_request:"..tostring(_res).." ==> ".._err)
```

Comment:
@pestOO I think that current version is more readable. I don't think that we should minimise unit testing code in prejudice of simplicity